### ISSUE TITLE
Amphetamine not launching at login

### ISSUE THREAD

**Issue by cl2solutions** [x74353/AmphetamineBeta#18]
By using the Preference within Amphetamine to launch at login, the app fails to launch at login and removes the checked box from preferences.

**Comment by htismaqe** [comment]
I have the same issue after upgrading to MacOS Ventura.  It worked fine on previous MacOS versions.

**Comment by Jeffofmany** [comment]
Thats exactly my problem too, sure hope it gets fixed.

**Comment by mzaur** [comment]
Same here on two separate macs. Still not fixed months later. I'm beginning to think this project has been abandoned.

**Comment by Jeffofmany** [comment]
I just looked at the About sheet on the app. It shows 5.2.2 and shows a copyright of 2023. Seems like it's still supported, but if more people would join this thread and state they have the problem too, it might get more attention?

**Comment by Jeffofmany** [comment]
I tried this:
https://iffy.freshdesk.com/support/solutions/articles/48000078422-amphetamine-does-not-launch-at-login
No fix
Upon checking the 'Launch Amphetamine at Login' box and logoff then login, the box was unchecked. I beleive this started when I upgraded to Ventura release.

**Comment by digitallawyer** [comment]
In case this helps someone. I was able to get Amphetamine to launch on startup by manually removing and then re-adding it from system settings > log in items on Ventura 13.2. Once Manually added to the list of apps under "Open at Login", Amphetamine starts upon launch.

![Screenshot 2023-02-20 at 9 42 13 AM](https://user-images.githubusercontent.com/1132237/220172254-d1d4ad6c-82de-4145-901c-38f1897169a7.png)

**Comment by htismaqe** [comment]
> In case this helps someone. I was able to get Amphetamine to launch on startup by manually removing and then re-adding it from system settings > log in items on Ventura 13.2. Once Manually added to the list of apps under "Open at Login", Amphetamine starts upon launch.
> 
> ![Screenshot 2023-02-20 at 9 42 13 AM](https://user-images.githubusercontent.com/1132237/220172254-d1d4ad6c-82de-4145-901c-38f1897169a7.png)

I was able to do the same.

**Comment by Jeffofmany** [comment]
This worked for me too! In my case I had to delete the Amphetimine Lanch Helper from Open At Login list first then manually add as described by digitallawer. Nice workaround, thanks

**Comment by aegjoyce** [comment]
I tried everything on the help page, all the terminal commands, uninstalling and reinstalling Amphetamine - none of it worked, just wouldn't launch at startup. So I opened the package contents and deleted AmphetamineLoginHelper and then manually added the Amphetamine app to login items. It works now! I don't know what the helper is doing but it doesn't help!

**Comment by htismaqe** [comment]
Yeah, I can't say for sure but I'd bet a large sum that they changed the system permissions and routines in Ventura just enough to break the Helper App.  This happens more often than one might realize as Apple releases security fixes and OS patches.

**Comment by x74353** [comment]
Hello all, Amphetamine 5.2.3 should hopefully fix this issue. You can test out the beta here: testflight.apple.com/join/GeZMOAsG

**Comment by digitallawyer** [comment]
Thank you for the update - all the hard work on this is much appreciated.

**Comment by Jeffofmany** [comment]
When I installed 5.2.3 it replaced the old app (5.2.2). When I ran the 5.2.3 I verified the version number in the ‘About’ in its menu. I have now deleted it all together. I also have Amphetamine Enhancer installed, could that cause a problem? I will uninstall it too. Then I can reinstall the beta version and test. If you want I can delete traces in the system I.E Library folder etc. I will wait for your reply.

**Comment by x74353** [comment]
@Jeffofmany No, the Enhancer shouldn't cause an issue. If the launch at login issue is persisting, try manually removing Amphetamine from the Login Items in System Settings (if it's there). You can also try deleting all of Amphetamine's Preferences/Setting via Terminal: `defaults delete com.if.Amphetamine`
